yeonjae/programmers-2/K진수에서소수개수구하기.js

```diff
@@ -0,0 +1,30 @@
+function isPrime(number) {
+  if (number <= 1) {
+    return false;
+  }
+
+  for (let i = 2; i <= Math.sqrt(number); i++) {
+    if (number % i === 0) {
+      return false;
+    }
+  }
+  return true;
+}
+
+function solution(n, k) {
+  const binaryNumArr = n.toString(k).split("0");
+  let result = 0;
+
+  for (let i = 0; i < binaryNumArr.length; i++) {
+    if (isPrime(+binaryNumArr[i])) {
+      result++;
+    }
+  }
+
+  return result;
+}
```

Comment:
```suggestion
function isPrime(number) {
  if (number <= 1) {
    return false;
  }

  for (let i = 2; i <= Math.sqrt(number); i++) {
    if (number % i === 0) {
      return false;
    }
  }
  return true;
}

function solution(n, k) {
  const kDecimalNumArr = n.toString(k).split("0").map(Number);
  let result = 0;

  for (let i = 0; i < kDecimalNumArr.length; i++) {
    if (isPrime(binaryNumArr[i])) {
      result++;
    }
  }

  return result;
}
```
- 깔끔한 풀이 잘봤습니다! 🙇‍♂️
- binary는 2진을 의미하기때문에 DeepL에서 k진수를 번역했을 때 k-decimal이 나오기 떄문에 decimal로 바꿔봤습니다.
- 또한, NumArr인데 사실 split후에 어떠한 형변환을 해주지 않았기 때문에 해당 요소는 문자열이라 변수 네이밍에 맞게 map을 통한 형변환작업까지해서 리팩토링 진행해봤습니다!

Reply:
아 바이너리가 진수를의미한다고 생각했는데 2진수였군요,,,! 코멘트 감사합니다🙇🏻‍♀️🙇🏻‍♀️